docs/getting_started/how_to_install_mautic.rst

```diff
@@ -249,7 +249,57 @@ You can now login to your Mautic instance with your Mautic Admin credentials.
 
 Installing from GitHub
 **********************
-Todo
+
+It's essential to have all the files locally - including automated tests - from the GitHub repository when testing Mautic or contributing to it. Many of these files aren't included in the production build process.
+
+Cloning Mautic from GitHub
+==========================
+
+1. Install the :xref:`GitHub CLI` tool.
+
+2. Click **Fork** at the top-right corner of the Mautic GitHub repository page to make a personal fork. You can also click to go directly to your fork if you already have one or create one if you don't.
+
+3. After the fork is complete, click the green **Code** button to access the command for cloning the repository.
+
+4. Switch to your terminal, and when in the directory where you wish to install Mautic, paste the command using the :xref:`GitHub CLI` tool this is in the format:
+
+.. code-block:: shell
+  
+  gh repo clone username/mautic
+
+.. note:: 
+
+  Please always choose to fork into a **personal account** rather than an organization. The latter prevents Mautic's maintainers from working with your Pull Request.
+
+Install Mautic using DDEV
+==========================
+
+You can use :xref:`DDEV` which is recommended for testing and development with Mautic. To get started:
+
+
+#.  Install :xref:`DDEV`.
+
+#.  Install and ensure you have :xref:`Docker` running on your system.
+
+#.  You can now change into the Mautic directory and kick off the DDEV quickstart using the command:
+   
+.. code-block:: shell
+  
+    ddev start
+
+.. note:: 
+
+  For troubleshooting ``ddev start``, paste ``ddev config global --nfs-mount-enabled=false && ddev config --nfs-mount-enabled=false`` and try again. For steps on DDEV troubleshooting, See :xref:`DDEV Troubleshooting`.
```

Comment:
```suggestion
  For troubleshooting ``ddev start``, see :xref:`DDEV Troubleshooting`.
```
I still feel we should not be providing such a specific command. Folks may not have the issue this command fixes. We should send them instead to the DDEV documentation.

Reply:
Yeah, this works